### PR TITLE
feat(text-area): add a way to manually set text-area in error state (VIV-990)

### PR DESCRIPTION
**Pull request by olaf-k**
pretty much a copy/paste of #1103

### CODE REVIEW

File: libs/components/src/lib/text-area/text-area.spec.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ describe('vwc-text-area', () => {
 	});
 
 	describe('basic', () => {
-		it('should be initialized as a vwc-text-field', async () => {
+		it('should be initialized as a vwc-text-area', async () => {
```
Comment (YonatanKra) [inline libs/components/src/lib/text-area/text-area.spec.ts:41]: :)

File: libs/components/src/lib/text-area/text-area.spec.ts [hunk 2]
```diff
@@ -448,6 +448,61 @@ describe('vwc-text-area', () => {
...
+		});
+
+		it('should display the given error message', async function () {
+			element.errorText = forcedErrorMessage;
+			await elementUpdated(element);
+			const errorElement = element.shadowRoot?.querySelector('.error-message');
+			expect(errorElement !== null).toBeTruthy();
+		});
+
+		it('should replace/restore the current error state, if any, when set/removed', async function () {
```
Comment (YonatanKra) [inline libs/components/src/lib/text-area/text-area.spec.ts:477]: Please split this to two different `it`s.

**Review by YonatanKra — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/text-area/text-area.spec.ts (hunk 2 above)
Comment (olaf-k) [reply libs/components/src/lib/text-area/text-area.spec.ts:477]: we had the same exchange in https://github.com/Vonage/vivid-3/pull/1103#discussion_r1170161786
how would you split the test?

File: libs/components/src/lib/text-area/text-area.spec.ts [hunk 3]
```diff
@@ -448,6 +448,61 @@ describe('vwc-text-area', () => {
...
+			expect(element.validationMessage).toBe(forcedErrorMessage);
+			expect(element.validity.valid).toBeFalsy();
+
+			element.errorText = '';
+			await elementUpdated(element);
+
+			expect(element.validationMessage).toBe(initialErrorMessage);
+			expect(element.validity.valid).toBeFalsy();
+		});
+	});
```
Comment (YonatanKra) [inline libs/components/src/lib/text-area/text-area.spec.ts:504]: Something in the spirit of:
```suggestion
		it('should replace the current error state when set', function () {
		element.required = true;
		setToBlurred();
		element.errorText = forcedErrorMessage;
		await elementUpdated(element);

		expect(element.validationMessage).toBe(forcedErrorMessage);
	});
	
	it('should restore the current error state when removed', async function () {
		element.required = true;
		setToBlurred();
		await elementUpdated(element);
		const initialErrorMessage = element.validationMessage;
		element.errorText = forcedErrorMessage;
		await elementUpdated(element);

		element.errorText = '';
		expect(element.validationMessage).toBe(initialErrorMessage);
	});
```

**Review by YonatanKra — APPROVED** [review]
LGTM.
image

**Pull request merged**